doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -1,14 +1,16 @@
 [id='offline-repo-proc']
-= Preparing a Maven repository for offline use 
+= Preparing a Maven mirror repository for offline use 
 
-If your {OPENSHIFT} environment does not have outgoing access to the public Internet, you must prepare a Maven repository with a mirror of all the necessary artifacts and make this repository available to your environment for use in source to image (S2I) builds.
+If your {OPENSHIFT} environment does not have outgoing access to the public Internet, you must prepare a Maven repository with a mirror of all the necessary artifacts and make this repository available to your environment.
 
 Skip this procedure if your {OPENSHIFT} environment is connected to the Internet.
 
 .Procedure
 
-. Build the source of your services on any machine using the `mvn clean install` command.
+. Prepare a Maven release repository to which you can write. The repository must allow read access without authentication. Your OpenShift environment must have access to this repository. You can deploy a Nexus repository manager in the OpenShift environment; for instructions, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/tutorials#nexus-setting-up-nexus[Setting up Nexus]. 
```

Comment:
Suggest "... environment. For instructions on / details about using Nexus with OpenShift, ..." (or whatever). Same rationale as I mentioned else in a similar case.